roles/kubernetes/client/tasks/main.yml

```diff
@@ -86,7 +69,7 @@
     user_certs: "{{ admin_kubeconfig['users'][0]['user'] }}"
     username: "kubernetes-admin-{{ cluster_name }}"
     context: "kubernetes-admin-{{ cluster_name }}@{{ cluster_name }}"
-    override_cluster_name: "{{ { 'clusters': [ { 'cluster': cluster_infos, 'name': cluster_name } ] } }}"
+    override_cluster_name: "{{ { 'clusters': [ { 'cluster': (cluster_infos|combine({'server': 'https://'+external_apiserver_address+(external_apiserver_port|string)})), 'name': cluster_name } ] } }}"
```

Comment:
It makes an invalid URI that lacks a delimiter (:) between external_apiserver_address and external_apiserver_port.